daemon/cmd/daemon.go

```diff
@@ -315,7 +316,13 @@ func newDaemon(ctx context.Context, cleaner *daemonCleanup, params *daemonParams
 	}
 
 	ctmap.InitMapInfo(option.Config.EnableIPv4, option.Config.EnableIPv6, option.Config.EnableNodePort)
-	policymap.InitMapInfo(option.Config.PolicyMapEntries)
+
+	nCPU, err := ebpf.PossibleCPU()
```

Comment:
If you make the policy stat map a cell, you could put this into its start hook and avoid further polluting `daemon.go`.

Reply:
Done.